### ISSUE TITLE
Autopilot reacting too slow to speed changes (11.40)

### ISSUE THREAD

**Issue by Bill-Clinton69** [magknight/787aviatorsTracker#118]
When climbing and setting a speed of 290 the plane deviates +/- 20 knots before finally settling down. 

https://gyazo.com/e50b8407d561600e339ba18b96eda6c4
https://gyazo.com/9de84473d73c41ead76fbcf833df9bd5
https://gyazo.com/f827c2516df668d242f72044ae3b2447

Big issue after take off and accelerating to 250 knots. The plane will reach up 270 knots then slows down to 240 knots before stabilizing to 250. In my opinion the autopilot reacts too slow to speed changes.

**Comment by aeroniemi** [comment]
so it's an auto throttle issue, or a autopilot (flch) issue?

**Comment by Bill-Clinton69** [comment]
Flch issue. I was using vnav to maintain 290 knots for climb.

**Comment by Bill-Clinton69** [comment]
OS: Windows 10
X-Plane version: XP 11.40b6, experimental flight model on
Version 1.4.0-devBeta3

**Comment by aeroniemi** [comment]
OK, so it's likely an issue as a result of you using 11.40, which isn't supported by the shipped flight model

**Comment by Bill-Clinton69** [comment]
Roger. Another problem I have is that autothrottle will maintain maximum thrust during cruise. Should I open another issue?  The autothrottle will not decrease to maintain the assigned mach 0.8 speed. 

![image](https://user-images.githubusercontent.com/56235285/66414529-dddeb780-e9f9-11e9-9240-33b35babdcdd.png)

So when I set CRZ thrust the N1 stays at 96.6 (max). If I set to cont I have the same issue.

![image](https://user-images.githubusercontent.com/56235285/66414576-fcdd4980-e9f9-11e9-927d-78a6c7668569.png)
